qna/DE-receipt-handout.md

```diff
@@ -0,0 +1,13 @@
+## Question
+
+What should i do when the order is processed but the posreceipt can't be print because of an internal technical reason of the POS?
+
+## Metadata tags
+
+lang-en, market-de, market-at, market-fr, middleware, PosCreator, 
+
+## Answer
+
+The process of printing and handing out receipts is not covered by our solution. We assume that once the order is beeing processed the receipt is beeing handed out. Therefore there is no requirement from our side to log that issue into our journal.
```

Comment:
```suggestion
The process of printing and handing out receipts is not covered by our solution. We assume that once the order is beeing processed and the invoice is send to fiskaltrust.Middleware the receipt is being handed out. Therefore there is no requirement from our side to log that issue into our journal.
```

Reply:
What about market-de ... when the order is processed ... then the transaction is started but no invoice is processed and the transaction should be closed? voided? ...

Reply:
Comment: This qna is only related to DE.

"The obligation to issue receipts to the customer (§ 146a Abs. 2 AO) is a legal requirement which must be fulfilled by the POS system itself.
The process of printing and handing out receipts is not part of the fiskaltrust compliance. We assume that once the order is beeing processed and the receipt is send to fiskaltrust.Middleware, the receipt is being handed out as required. Therefore there is no requirement to log that issue into our journal by the fiskaltrust.Middleware."